airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -129,8 +129,7 @@ public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws Retr
                 ChildWorkflowOptions.newBuilder()
                     .setWorkflowId("sync_" + maybeJobId.get())
                     .setTaskQueue(TemporalJobType.CONNECTION_UPDATER.name())
-                    // This will cancel the child workflow when the parent is terminated
-                    .setParentClosePolicy(ParentClosePolicy.PARENT_CLOSE_POLICY_TERMINATE)
+                    .setParentClosePolicy(ParentClosePolicy.PARENT_CLOSE_POLICY_REQUEST_CANCEL)
```

Comment:
Added some clarifying comments.

Reply:
is the idea that sending the cancel signal means that we can retry the launcher activity? terminate means it cannot be retried. 

cancelling the launcher activity does not cancel its children, right? because when we resume we are reconnecting to the children. it just kills the launcher activity so it can be retried subsequently.

Reply:
We do want cancellation to cancel its children. Otherwise, cancellation would not be able to stop a sync in progress. The current `LauncherWorker` implementation kills everything for the connection when cancelled. This flag is necessary for this behavior; otherwise the cancellation doesn't propagate and the async kubernetes process is orphaned forever.

Reply:
I'll add info from our last two messages in the comments, but the behavior is correct imo.

Reply:
cool. based on added info. i agree.